src/extensions/cp/app/menu.lua

```diff
@@ -757,7 +616,7 @@ function menu.mt:doFindMenuUI(path, options)
                                             if title == nil then
                                                 error(format("Unexpected `nil` menu item title while searching for '%s'", currentTitle))
                                             end
-                                            return exactMatch(title, currentTitle, options.plain)
```

Comment:
@randomeizer - Ok, so this is where things are failing.

If our test code is: `require("cp.apple.finalcutpro"):doSelectMenu({"Edit", "Undo.*"}):Now()`

...then at this point `title` is "Undo Move" (i.e. whatever appears in the actual FCPX menubar interface), and `currentTitle` is just "Undo". The reason it's failing is because `currentTitle` is lacking the pattern matching text, so `exactMatch()` returns `false`.

Replacing `currentTitle` with `menuItemName` fixes this issue - at least in this case. I'm just not exactly sure if this is a proper fix or not? Thoughts?